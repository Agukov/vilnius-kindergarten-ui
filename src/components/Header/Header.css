.underlineHeader{
  border-bottom: 1px solid black;
}

.headerText{
  color: red;
}

#headeris{
  border-bottom: 1px solid red;
  color: black;
}
