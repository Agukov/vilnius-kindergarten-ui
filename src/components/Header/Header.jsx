import React from 'react'
import propTypes from 'prop-types'
import styles from './header.css'
import {PageHeader} from 'react-bootstrap'

export class Header extends React.Component {
  render(){
      return (
        <PageHeader>
          {this.props.name}
          <br/>
          <small className={styles.subHeader}>
            Web App for some lyrics
          </small>
        </PageHeader>
      )
  }
}

Header.defaultProps = {
  name: 'World'
}

Header.propTypes = {
  name : propTypes.string
}
