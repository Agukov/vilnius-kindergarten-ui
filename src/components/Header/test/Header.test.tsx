import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { Header } from '../Header';
import 'jest-enzyme';

describe('Header Component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Header title="header"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
