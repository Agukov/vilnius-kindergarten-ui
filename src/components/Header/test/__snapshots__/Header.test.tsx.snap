// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders correctly 1`] = `
<div
  className="App-header"
>
  <h1>
    header
  </h1>
  <h3>
    optional sub title
  </h3>
</div>
`;
